src/Widgets/calendar/ControlHeader.vala

```diff
@@ -19,37 +19,61 @@
 
 
 namespace DateTime.Widgets {
-    public class ControlHeader : Gtk.Box {
+    public class ControlHeader : Gtk.Grid {
         public signal void left_clicked ();
         public signal void right_clicked ();
         public signal void center_clicked ();
+
         public ControlHeader () {
```

Comment:
Since we changed this to a `Gtk.Grid` subclass, these constructor properties are redundant: `Gtk.Grid` is already horizontal by default. So this whole constructor can be removed